pkg/api/dtos/alerting.go

```diff
@@ -77,6 +77,24 @@ type AlertNotification struct {
 	Settings              *simplejson.Json `json:"settings"`
 }
 
+func NewAlertNotificationChannel(notification *models.AlertNotification) *AlertNotificationChannel {
+	return &AlertNotificationChannel{
+		Id:        notification.Id,
+		Uid:       notification.Uid,
+		Name:      notification.Name,
+		Type:      notification.Type,
+		IsDefault: notification.IsDefault,
+	}
+}
+
+type AlertNotificationChannel struct {
```

Comment:
Created a new struct instead of reusing the `AlertNotification` struct. Makes it more explicit and harder to make mistakes.

Reply:
not sure I like having AlertNotification and AlertNotificationChannel dtos that are the same thing without any clear way to know what makes them different. Easy to iintoduce mistakes / leaks in the future.
ideas:
* AlertNotificationChannelFull 
* AlertNotificationChannelStripped

* AlertNotificationChannelForEditor
* AlertNotificationChannelForViewer

Reply:
Given https://github.com/grafana/grafana/pull/18422#discussion_r311410954 I would rename this to AlertNotificationLookup.

Reply:
yes!